### PR TITLE
[fix] Invalid even more cookies

### PR DESCRIPTION
**Pull request by SilverViper**
See: #162

### CODE REVIEW

File: helpers.lua [hunk 1]
```diff
@@ -214,7 +214,7 @@ function delete_redirect_cookie()
                        "; Expires="..expired_time..
                        "; Secure"..
                        "; HttpOnly"..
-                       "; SameSite=Lax ;;"
+                       "; SameSite=Lax"
     ngx.header["Set-Cookie"] = "SSOwAuthRedirect=;" ..cookie_str
```
Comment (kay0u) [inline helpers.lua]: > Syntax error in Set-Cookie: SSOwAuthRedirect=;; Path=/yunohost/sso/; Expires=Thu, 01 Jan 1970 00:00:00 UTC; Secure; HttpOnly; SameSite=Lax ;; at position 18.

I think we should remove this `;` here (or the one line 213 : `local cookie_str = "; Path="..conf["portal_path"]..`).

**Comment by kay0u** [comment]
What if we left only one `;`?

**Comment by SilverViper** [comment]
> What if we left only one ;?

Response headers do not require trailing ;
[https://developer.mozilla.org/en-US/docs/Glossary/Response_header](url)

**Comment by alexAubin** [comment]
Bumpity bump, anybody has some insight on this ? Shall we yolomerge and see what happens lulz ?

**Comment by alexAubin** [comment]
Sounds like those ;; were initially added by me in https://github.com/YunoHost/SSOwat/commit/2699aa8db727c20dd74fe05887c511fa3f1cf68a but I have no idea why I did this ...

Anyway, yolomerging !

**Pull request merged**